src/Elcodi/Admin/PageBundle/Controller/PageController.php

```diff
@@ -189,12 +191,17 @@ public function editAction(
      */
     public function enableAction(
         Request $request,
-        EnabledInterface $entity
+        EnabledInterface $page
     )
     {
+        if ($page->isPersistent()) {
```

Comment:
"This check should go in `Page::setEnable`" — says the old developer in me, while he can't get this idea off from his head.
Then he remembers we're being anemic here, and throws himself from a cliff.